content/pl/docs/tasks/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Zadania
+main_menu: true
+weight: 50
+content_template: templates/concept
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+
+W tej części dokumentacji Kubernetesa znajdują się opisy
+sposobu realizacji różnych zadań. Podane są one zazwyczaj jako
+krótka sekwencja kilku kroków związanych z pojedynczym zadaniem.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Graficzny interfejs użytkownika _(Dashboard)_
+
+Instalacja i użycie graficznego interfejsu użytkownika z poziomu przeglądarki do zarządzania aplikacjami w kontenerach uruchomionymi na klastrze Kubernetes.
+
+## Jak używać polecenia kubectl
+
+Instalacja i konfiguracja polecenia `kubectl` do bezpośredniego zarządzania klastrami Kubernetes.
+
+## Konfigurowanie podów i kontenerów
+
+Najpopularniejsze czynności związane z konfiguracją podów i kontenerów.
+
+## Uruchamianie aplikacji
+
+Standardowe metody zarządzania aplikacjami, m. in. prowadzenie płynnych aktualizacji _(rolling updates)_, przekazywanie konfiguracji do podów oraz skalowanie horyzontalne podów.
+
+## Uruchamianie zadań
+
+Uruchamianie zadań w trybie równoległym.
+
+## Dostęp do aplikacji na klastrze
+
+Rozkładanie obciążenia i przekierowanie ruchu sieciowego, konfigurowanie firewalla i usług DNS w celu zapewnienia dostępu do aplikacji w klastrze.
+
+## Monitoring, rejestracja i znajdowanie błędów
+
+Konfigurowanie monitoringu oraz rejestrowanie zdarzeń i komunikatów, pomagające rozwiązywać problemy związane z pracą klastra lub aplikacji uruchamianych w kontenerach.
+
+## Dostęp do API Kubernetes
+
+Różne metody bezpośredniego dostępu do API Kubernetes.
+
+## Używanie TLS
+
+Konfigurowanie aplikacji w taki sposób, aby korzystała i ufała łańcuchowi certyfikatów wydawanych przez główne centrum autoryzacji (CA) klastra.
+
+## Administracja klastrem
+
+Standardowe metody zarządzania klasterem.
+
+## Administracja federacją
+
+Konfigurowanie federacji klastrów.
+
+## Zarządzanie aplikacjami ze stanem (_Stateful_)
+
+Popularne zadania związane z zarządzaniem aplikacjami ze stanem _(Stateful)_, w tym: skalowanie, usuwanie i rozwiązywanie problemów dotyczących _StatefulSets_.
```

Comment:
```suggestion
Popularne zadania związane z zarządzaniem aplikacjami stanowymi _(Stateful)_, w tym: skalowanie, usuwanie i rozwiązywanie problemów dotyczących _StatefulSets_.
```

Reply:
I believe we use "ze stanem" elsewhere.

Reply:
I was not sure about it...

Reply:
I'll keep as it is. There is an inconsistency here. I'll create another PR for that.